electroncash_gui/qt/main_window.py

```diff
@@ -231,6 +233,7 @@ def add_optional_tab(tabs, tab, icon, description, name, default=True):
         self.gui_object.update_available_signal.connect(self.on_update_available)  # shows/hides the update_available_button, emitted by update check mechanism when a new version is available
         self.gui_object.new_window_initialized_signal.connect(self.new_window_initialized)
         self.history_list.setFocus(True)
+        self.token_history_list.setFocus(True)
```

Comment:
Probably you were just doing everything history_list did here.. and weren't aware of what `setFocus` does..

`setFocus` grabs keyboard focus .. considering that the history_list is up and not the token_history_list.. this line should be removed.

```suggestion
```

Reply:
Ah ok, and indeed, mostly this is based on history_list!